lib/Conch/Controller/DeviceReport.pm

```diff
@@ -267,11 +270,11 @@ sub _record_device_report {
 			}
 
 			$dr->{disks}
-				and $log->info("Recorded disk info for Device $device_id");
+				and $log->info("Recorded disk info for Device $device->id");
```

Comment:
This will not render correctly. use `$device_id`.

Reply:
Addressed